packages/cli/src/oclif/commands/build.js

```diff
@@ -0,0 +1,55 @@
+const BaseCommand = require('../ZapierBaseCommand');
+const { buildFlags } = require('../buildFlags');
+const { flags } = require('@oclif/command');
+const {
+  BUILD_PATH,
+  SOURCE_PATH,
+  CURRENT_APP_FILE
+} = require('../../constants');
+
+const { buildAndOrUpload } = require('../../utils/build');
+
+class BuildCommand extends BaseCommand {
+  async perform() {
+    this.log('Building Project\n');
+    await buildAndOrUpload(
+      { build: true },
+      {
+        skipNpmInstall: this.flags['skip-npm-install'],
+        disableDependencyInjection: this.flags['disable-dependency-injection']
```

Comment:
This was named `disable-dependency-detection`. Was it renamed on purpose? I think the original name is more accurate.

Reply:
whoops, definitely an accident.